CIE-5615 - Permission error using c8yedge installer (#4750)
SoftwareAG/c8y-docs: content/change-logs/edge/edge-2026.0.1-permission-error-in-c8yedge-install.md

---
date: 2026-01-02
title: c8yedge install no longer fails on systems with hardened /tmp configuration
change_type:
  - value: change-VSkj2iV9m
    label: Fix
product_area: Edge
component:
  - value: component-g7hnfbu4J
    label: Edge
build_artifact:
  - value: tc-nJH2U7g3u
    label: edge
ticket: CIE-5615
version: 2026.0.1
---

> In previous Edge releases, the **c8yedge** installation could fail during K3s installation on systems where the `/tmp` directory was hardened with the `noexec` mount option. In such environments, the installation failed with a permission denied error when attempting to execute the K3s installation script from the `/tmp` directory. This issue has been resolved, and the installation now works correctly on systems with a hardened `/tmp` configuration.

Reviewer: Now that I've read all of the guidelines about modal verbs and past/future tense :-D

```suggestion
In previous Edge releases, the **c8yedge** installation fails during K3s installation on systems where the `/tmp` directory is hardened with the `noexec` mount option. In such environments, the installation fails with a permission denied error when attempting to execute the K3s installation script from the `/tmp` directory. This issue has been resolved, and the installation now works correctly on systems with a hardened `/tmp` configuration.
```
